build/BranchInfo.Stable.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <MajorVersion>1</MajorVersion>
+    <MinorVersion>13</MinorVersion>
```

Comment:
should it be in sync with BranchInfo.props?

Reply:
good point. to keep as same value for now.